bin/generate

```diff
@@ -142,11 +142,11 @@ function check_files_and_commit() {
   # Check that base branch files are unchanged. Error if they are.
   # Add the new files and commit.
 
-  if ! git diff-index --quiet HEAD;
-  then
-    git diff-index HEAD | cut -f 2 | xargs -I % echo "Error: lookml-generator modified %"
-    exit 1
-  fi
+  # if ! git diff-index --quiet HEAD;
+  # then
+  #   git diff-index HEAD | cut -f 2 | xargs -I % echo "Error: lookml-generator modified %"
+  #   exit 1
+  # fi
```

Comment:
There are a couple of [views in lookerhub for firefox_desktop](https://github.com/mozilla/looker-hub/tree/main/firefox_desktop/views) that have been created manually. Before merging this PR, the views need to be deleted. We'll only use the auto-generated desktop views in the future.